Integrations/integration-Zscaler.yml

```diff
@@ -99,18 +98,171 @@ script:
         cmd_url = '/authenticatedSession'
         result = http_request('DELETE', cmd_url, None, DEFAULT_HEADERS)
 
-    def blacklist_url(url):
-        cmd_url = '/security/advanced/blacklistUrls?action=ADD_TO_LIST'
+    def update_blacklist(url, action, ioc_type):
```

Comment:
don't do this flags/if/else thing. split the function to 2 separate functions. add url to blacklist and remove from list